src/nxdoc/nuxeo-server/installation/nuxeo-and-redis.md

```diff
@@ -195,6 +195,14 @@ The old Redis Transient Store can still be used with `nuxeo.transientstore.provi
 
 ## Clean-up
 
+It could happen that because of a Nuxeo cluster failure, Redis is keeping corrupted data. The following commands will help you restore a stable state.
+
+{{#> callout type='warning' }}
+In the following commands, the `nuxeo` prefix is used. If a different value is set in `nuxeo.conf`, `nuxeo.redis.prefix`, consider updating the code samples accordingly.
```

Comment:
In the following Lua scripts, the `nuxeo:` prefix is used for Redis keys.
    If a different value is set for `nuxeo.redis.prefix` in `nuxeo.conf`,
    you should update the code samples accordingly.